protocol/messenger_communities.go

```diff
@@ -347,6 +347,26 @@ func (m *Messenger) SetMuted(communityID types.HexBytes, muted bool) error {
 	return m.communitiesManager.SetMuted(communityID, muted)
 }
 
+func (m *Messenger) MuteCategory(communityID string, categoryID string, muted bool) error {
```

Comment:
`categoryID` is passed in as a parameter but it isn't used in this function.

Reply:
fixed :)